src/dialects/oracle/schema/columncompiler.js

```diff
@@ -26,14 +26,14 @@ assign(ColumnCompiler_Oracle.prototype, {
     });
   },
 
-  increments () {
+  increments ({ primaryKey = true } = {}) {
     this._createAutoIncrementTriggerAndSequence();
-    return 'integer not null primary key';
+    return primaryKey ? 'integer not null primary key' : 'integer not null';
```

Comment:
I'm afraid I'm not quite sure where to start on this one, as I need to create a separate sequence using a namespace and reference it here.

https://www.w3schools.com/sql/sql_autoincrement.asp